tempest/api/workloadmgr/barbican/test_barbican.py

```diff
@@ -2340,3 +2340,130 @@ def test_5_barbican(self):
             reporting.set_test_script_status(tvaultconf.FAIL)
         finally:
             reporting.test_case_to_write()
+
+    @decorators.attr(type='workloadmgr_cli')
+    def test_6_barbican(self):
+        try:
+            test_var = "tempest.api.workloadmgr.barbican.test_"
+            tests = [[test_var + "create_workload_with_encryption_CLI", 0],
+                     [test_var + "Create_Multiple_workloads_with_same_Secret_UUID", 0],
+                     [test_var + "create_workload_with_wrong_secretUUID", 0]]
+            reporting.add_test_script(tests[0][0])
+            self.kp = self.create_key_pair(tvaultconf.key_pair_name)
+            self.vm_id = self.create_vm(key_pair=self.kp)
+
+            self.secret_uuid = self.create_secret()
+
+            # Create workload with CLI
+            workload_create_with_encryption = command_argument_string.workload_create_with_encryption + \
+                                              " --insecure --instance instance-id=" + str(self.vm_id) + \
+                                              " --secret-uuid " + str(self.secret_uuid)
+            error = cli_parser.cli_error(workload_create_with_encryption)
+            if error and (str(error.strip('\n')).find('ERROR') != -1):
+                LOG.debug("workload with encryption creation unsuccessful : " + error)
+                reporting.add_test_step(
+                    "Execute workload_create_with_encryption command",
+                    tvaultconf.FAIL)
+                raise Exception(
+                    "Workload with encryption creation Failed")
+            else:
+                LOG.debug("workload with encryption created successfully")
+                reporting.add_test_step(
+                    "Execute workload_create_with_encryption command",
+                    tvaultconf.PASS)
+                time.sleep(10)
+                self.wid1 = query_data.get_workload_id_in_creation(
+                    tvaultconf.workload_name)
+                workload_available = self.wait_for_workload_tobe_available(
+                    self.wid1)
+                LOG.debug("Workload ID: " + str(self.wid1))
+
+                # Show workload details using CLI command
+                rc = cli_parser.cli_returncode(
+                command_argument_string.workload_show + self.wid1 + " --insecure")
```

Comment:
--insecure need not be added to the command here. It gets added to the command_argument_string.py file when fetch_resources.sh is run.

Reply:
removed